application_base.cpp

```diff
@@ -386,6 +373,9 @@ bool application_base::initialize_impl(int argc, char** argv, vector<abstract_pl
    std::string plugin_name;
    auto error_header = [&]() { return std::string("appbase: exception thrown during plugin \"") + plugin_name + "\" initialization.\n"; };
 
+   // setup handling of SIGINT/SIGTERM/SIGPIPE/SIGHUP during initialize
```

Comment:
I suppose this comment should have `SIGHUP` removed because `setup_signal_handling_on_ioc()` doesn't handle `SIGHUP` now